service/dap/types.go

```diff
@@ -14,13 +14,34 @@ func isValidLaunchMode(mode string) bool {
 	return false
 }
 
+// Default values for Launch/Attach configs.
+// Used to initialize configuration variables before decoding
+// arguments in launch/attach requests.
+var (
+	defaultLaunchAttachCommonConfig = LaunchAttachCommonConfig{
+		Backend:         "default",
+		StackTraceDepth: 50,
```

Comment:
We don't need defaultArgs in server.go anymore, do we?

Reply:
NewServer uses defaultArgs. Not sure if it does anything useful but I feel that the cleanup belongs to a separate change.

Reply:
NewServer uses them just to have defaults in case user supplies nothing. Your new code accomplishes that now instead, so the original defaults should no longer be needed. I think this is very much in scope for this change. But at the very least please add a TODO.

Reply:
Added TODO. Please do the cleanup as you wish. Thanks.